boxsdk/object/trash.py

```diff
@@ -101,14 +113,43 @@ def get_items(self, limit: Optional[int] = None, offset: Optional[int] = None, f
             The offset of the item at which to begin the response.
         :param fields:
             List of fields to request.
+        :param sort:
```

Comment:
Maybe lets keep the orgde of params the same as for folder.get_items()
` marker: Optional[str] = None,
            use_marker: bool = False,
            sort: Optional[str] = None,
            direction: Optional[str] = None,`

Except `fields` of course in order not to make breaking change